bot/views.py

```diff
@@ -24,7 +24,7 @@ def index(request):
 @csrf_exempt
 def hook(request):
     blob = json.loads(request.body)
-    logging.info(blob)
+    logging.info(f'HOOK> {request.body.decode("utf-8")}')
```

Comment:
the logging is still pretty amateur, but here and below it's at least a little more descriptive.